.github/workflows/build-artifact.yml

```diff
@@ -33,12 +33,24 @@ jobs:
         sha=$(echo ${{ github.sha }} | cut -c 1-8)
         echo "::set-env name=artifact_sha::$sha"
 
+    - name: Check dependencies cache
+      id: cache-dependencies
+      uses: actions/cache@v2
+      with:
+        path: ./plugin/ThirdParty
+        key: ${{ runner.os }}-cactbot-${{ hashFiles('./util/fetch_deps.py', './util/DEPS.py') }}
+        restore-keys: |
```

Comment:
Looking at the [docs](https://docs.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows#example-using-multiple-restore-keys) they include the `${{ hashFiles }}` in their restore keys as the first hit. I haven't used this feature before, but is there any reason for us not to do the same?

Reply:
I think they write `hashFiles` in `restore-keys` for explaining the mechanic of cache matching.
As far as I saw in the [official examples](https://github.com/actions/cache/blob/main/examples.md) they don't do that.

Reply:
Interesting; the docs say they also pick the latest on everything that matches, so maybe they're relying on that mechanic. Nevermind!